site/src/components/mdx/mdx-page-content.tsx

```diff
@@ -54,17 +53,15 @@ export const MdxPageContent: FunctionComponent<MdxPageContentProps> = ({
         // if anchor is not empty, always allow scroll
         anchor !== "" ||
         // OR if previous path is either a docs or spec page
-        (previousRoute?.includes("/docs") &&
-          router.asPath?.includes("/docs")) ||
-        (previousRoute?.includes("/spec") && router.asPath?.includes("/spec"));
+        (previousRoute?.includes("/docs") && router.asPath?.includes("/docs"));
```

Comment:
Spec page is now at `/docs/spec`, so `router.asPath?.includes("/spec")` is not necessary